Changelog.md

```diff
@@ -1,3 +1,16 @@
+## 1.7.0 - Unreleased
+
+### Added
+
+- Samples for Nordic Thingy:91 X now also program the modem firmware package.
+
+### Changed
+
+- Modem firmware package is now programmed on devices only if not already
+  programmed.
+- Order of programming firmware for nRF91 Series devices. Application is now
+  programmed before modem firmware.
```

Comment:
```suggestion
```

We discussed it for thingy:91 X but I believe this is valid for all 91 series. @boundlesscalm should we keep change for the DKs?